index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
+    this.items.sort(function(a, b){return a-b});
+    this.length = this.items.length
+  }
 
-  get(pos) {}
+  get(i) {
+    let result = this.items[i]
+    if (result == undefined) {
+      throw new Error('OutOfBounds')
+    } else {
+      return result
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length != 0) {
+      return Math.max.apply(Math, this.items)
+    } else {
+      throw new Error('EmptySortedList')
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length != 0) {
+      return Math.min.apply(Math, this.items)
+    } else {
+      throw new Error('EmptySortedList')
+    }
+  }
 
-  sum() {}
+  sum() {
+    if (this.items.length != 0) {
+      return this.items.reduce((acc, cVal) => acc + cVal)
```

Comment:
Esto esta bien, pero te recomiendo que siempre lo pongas el valor predeterminado del reduce, es de buenas practicas hacerlo y además te va a funcionar mejor
```suggestion
      return this.items.reduce((acc, cVal) => acc + cVal, 0)
```